daemon/daemon.go

```diff
@@ -262,7 +262,7 @@ func (daemon *Daemon) restore() error {
 
 	for id, c := range containers {
 		group.Add(1)
-		go func(c *container.Container) {
+		go func(id string, c *container.Container) {
```

Comment:
looks like this was changed in https://github.com/moby/moby/pull/38301

/cc @cyphar

Reply:
Can we use `c.Id`?